typegen/generate-modules.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const path = require('path');
+
+function generateSubModules(mainJsFile, basePath) {
+  const pathMap = new Map();
+  const regex = /export\s+(\S+|\{[\s\w,]*\})\s+from\s+'(\.\/)?(\S+)'/g;
```

Comment:
Can you either write a test describing what this does or add some comments? Complex regex are tough to maintain without a little help

Reply:
I have retrieved generate-modules.js script. Now it uses runtime information of react-vis library instead of parsing index.js